WcaOnRails/db/structure.sql

```diff
@@ -677,6 +677,7 @@ CREATE TABLE `polls` (
   `created_at` datetime NOT NULL,
   `updated_at` datetime NOT NULL,
   `comment` text COLLATE utf8_unicode_ci,
+  `confirmed_at` datetime DEFAULT NULL,
```

Comment:
As discussed with you over chat, I believe this `confirmed_at` field should replace the existing `confirmed` field, rather than live alongside it.